### PR TITLE
Use tag:psacertified.org,2019:legacy#tfm instead of PSA_IOT_PROFILE_1

### LINKED ISSUE #142: The EAT media type for `PSA_IOT_PROFILE_1` is incorrect
`PSA_IOT_PROFILE_1` parses as a URI, but not as not an absolute URI.  

While there is no explicit requirement in EAT to make it absolute, the following observations hold:

1. The OID form is required to be absolute, and
1. The idea is to make the profile identifier unique (without an
explicit registration procedure),

So I think _the spirit_ is to have absolute URIs there.

We'll either drop the `PSA_IOT_PROFILE_1` media type completely or make a separate media type registration for it.

### PR DESCRIPTION
**Pull request by thomas-fossati**
Use the identifier suggested by Mathias.

Fix #142

### CODE REVIEW

**Review by SimonFrost-Arm — APPROVED** [review]

**Review by mathias-arm — APPROVED** [review]

**Comment by thomas-fossati** [comment]
Having second thoughts about the proposal in #142, shouldn't it be `tag:psacertified.org,2019:psa#legacy` instead?  That'd look more consistent with the recommendation we have for minting new profiles from the base URI.

**Comment by mathias-arm** [comment]
> Having second thoughts about the proposal in #142: shouldn't it be `tag:psacertified.org,2019:psa#legacy` instead? That'd look more consistent with the recommendation we have for minting new profiles from the base URI.

That looks better.

File: draft-tschofenig-rats-psa-token.md [hunk 1]
```diff
@@ -1001,7 +1001,7 @@ No new media type registration is requested.
 To indicate that the transmitted content is a PSA attestation token,
 applications can use the `application/eat+cwt` media type defined in
 {{EAT-MEDIATYPES}} with the `eat_profile` parameter set to
-`tag:psacertified.org,2023:psa#tfm` (or `PSA_IOT_PROFILE_1` if the token is encoded
+`tag:psacertified.org,2023:psa#tfm` (or `tag:psacertified.org,2019:legacy#tfm` if the token is encoded
```
Comment (thomas-fossati) [inline draft-tschofenig-rats-psa-token.md:1004]:
```suggestion
`tag:psacertified.org,2023:psa#tfm` (or `tag:psacertified.org,2019:psa#legacy` if the token is encoded
```

File: draft-tschofenig-rats-psa-token.md [hunk 2]
```diff
@@ -1013,7 +1013,7 @@ Content-Formats" registry {{IANA-CoAP-Content-Formats}}:
 * One for the `application/eat+cwt` media type with the `eat_profile` parameter
   equal to `tag:psacertified.org,2023:psa#tfm`
 * Another for the `application/eat+cwt` media type with the `eat_profile`
-  parameter equal to `PSA_IOT_PROFILE_1`
+  parameter equal to `tag:psacertified.org,2019:legacy#tfm`
```
Comment (thomas-fossati) [inline draft-tschofenig-rats-psa-token.md:1016]:
```suggestion
  parameter equal to `tag:psacertified.org,2019:psa#legacy`
```

File: draft-tschofenig-rats-psa-token.md [hunk 3]
```diff
@@ -1024,7 +1024,7 @@ The Content-Formats should be allocated from the Expert review range (0-255).
 *  Id: [[To-be-assigned by IANA]]
 *  Reference: {{&SELF}}
 
-*  Media Type: `application/eat+cwt; eat_profile="PSA_IOT_PROFILE_1"`
+*  Media Type: `application/eat+cwt; eat_profile="tag:psacertified.org,2019:legacy#tfm"`
```
Comment (thomas-fossati) [inline draft-tschofenig-rats-psa-token.md:1027]:
```suggestion
*  Media Type: `application/eat+cwt; eat_profile="tag:psacertified.org,2019:psa#legacy"`
```

**Comment by hannestschofenig** [comment]
I don't have strong opinions regarding the choice of the identifier. Hence, make the change.

**Review by hannestschofenig — APPROVED** [review]
No objections

**Pull request merged**